tests/L0/run_instance_norm_nvfuser/test_instance_norm_nvfuser.py

```diff
@@ -0,0 +1,102 @@
+import itertools
+import unittest
+
+import torch
+import torch.nn as nn
+
+import apex
+from apex.normalization import InstanceNorm3dNVFuser
+
+class TestInstanceNormNVFuser(unittest.TestCase):
+    dtype = torch.float
+    track_running_stats = False
+    channels_last = False
+    affine = False
+    batch_size = 5
```

Comment:
I think the one set shape test here hides the issue I posted above about the shape specific fusion definition vs absence of shape in cache key.

Reply:
Yeah, will add changing shape to the tests.

Reply:
Added changing shapes to tests in 716fbc2e419a8ca307016e7ed8fa29995fb82a3e